ambry-api/src/main/java/com.github.ambry/store/StoreStats.java

```diff
@@ -36,4 +36,11 @@
    * element is the valid data size
    */
   Pair<Long, Long> getValidSize(TimeRange timeRange) throws StoreException;
+
+  /**
+   * Get stats related information of the store as a {@link StatsSnapshot}.
```

Comment:
not "stats related information". It is the stats itself. May be you can word it as, "Fetches the stats for the store as {@link StatsSnapshot}"